ReactWindows/ReactNative/Modules/AccessibilityInfo/AccessibilityInfoModule.cs

```diff
@@ -0,0 +1,77 @@
+﻿using ReactNative.Bridge;
+using ReactNative.Modules.Core;
+using System.Collections.Generic;
+using Windows.UI.ViewManagement;
+
+namespace ReactNative.Modules.Accessibilityinfo
+{
+    class AccessibilityInfoModule : ReactContextNativeModuleBase
+    {
+        private readonly AccessibilitySettings _accessibility = new AccessibilitySettings();
+        private readonly UISettings _settings = new UISettings();
+
+        private string GetRgbaString(UIElementType type)
+        {
+            var color = _settings.UIElementColor(type);
+            return "rgba(" + color.R + "," + color.G + "," + color.B + "," + color.A + ")";
+        }
+
+        private IDictionary<string, string> GetHighContrastColors()
+        {
+            return new Dictionary<string, string>
+            {
+                {  "windowText", GetRgbaString(UIElementType.WindowText) },
+                {  "hotlight", GetRgbaString(UIElementType.Hotlight) },
+                {  "grayText", GetRgbaString(UIElementType.GrayText) },
+                {  "highlightText", GetRgbaString(UIElementType.HighlightText) },
+                {  "highlight", GetRgbaString(UIElementType.Highlight) },
+                {  "buttonText", GetRgbaString(UIElementType.ButtonText) },
+                {  "buttonFace", GetRgbaString(UIElementType.ButtonFace) },
+                {  "window", GetRgbaString(UIElementType.Window) },
+            };
+        }
+
+        public AccessibilityInfoModule(ReactContext reactContext)
+            : base(reactContext)
+        {
+
+        }
+
+        public override string Name => "AccessibilityInfo";
+
+        public override IReadOnlyDictionary<string, object> Constants
+        {
+            get
+            {
+                return new Dictionary<string, object>
+                {
+                    // TODO: It would be better to have a sync GethIsHighContrast,
+                    // but this is not supported by the framework at the moment.
+                    { "initialHighContrast", _accessibility.HighContrast },
+                };
+            }
+        }
+
+        public override void Initialize()
+        {
+            _accessibility.HighContrastChanged += OnHighContrastChanged;
```

Comment:
>_accessibility.HighContrastChanged += OnHighContrastChanged; [](start = 12, length = 60)

Should we do this in `Initialize` or in the constructor?

Reply:
Eric suggested to use `Initialize`.

Reply:
I see. What's the reasoning behind preferring `Initialize` over the constructor? When does `Initialize` get called relative to `Constants`?